packages/ckeditor5-image/src/imageresize/imageresizehandles.js

```diff
@@ -56,8 +56,8 @@ export default class ImageResizeHandles extends Plugin {
 		editingView.addObserver( ImageLoadObserver );
 
 		this.listenTo( editingView.document, 'imageLoaded', ( evt, domEvent ) => {
-			// The resizer must be attached only to images loaded by the `ImageInsert` or `ImageUpload` plugins.
-			if ( !domEvent.target.matches( 'figure.image.ck-widget > img' ) ) {
+			// The resizer must be attached only to images loaded by the `ImageInsert`, `ImageUpload` or `LinkImage` plugins.
+			if ( !domEvent.target.matches( 'figure.image.ck-widget > img, figure.image.ck-widget > a > img' ) ) {
```

Comment:
I know that I suggested this solution, but I'm wondering now if we could check if this `img` DOM node has a mapping to view and model elements. WDYT?

Reply:
Hmm, no idea what kind of consequences this change will have and whether it covers all cases...

Reply:
Hmm seems that `img` element itself has no mapping to model element. Ok let's leave it this way for now.